pkg/config/default-config.yml

```diff
@@ -33,8 +33,8 @@ defaultEnvironment:
   DATASTORE_DATABASE_PORT: 5432
   DATASTORE_DATABASE_NAME: openslides
   DATASTORE_DATABASE_USER: openslides
-  DATASTORE_DATABASE_PASSWORD: openslides
-  DATASTORE_DATABASE_PASSWORD_FILE: /run/secrets/datastore_postgres_password
+  DATASTORE_DATABASE_USER_FILE: /run/secrets/postgres_user
```

Comment:
the user will be provided plain text, see https://github.com/OpenSlides/openslides-datastore-service/pull/164